Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <WarningsAsErrors>IDE2000</WarningsAsErrors>
```

Comment:
Why?

Reply:
Because double blank lines cause the check in to fail, which is very annoying, so better catch this early

Reply:
But thats because of what you introduced with dotnet_style_allow_multiple_blank_lines_experimental = false:warning? Or does dotnet format even today complain without this setting?

Reply:
Yes, it failed already when dotnet format was running after check in, it seems to be a hard coded rule

Reply:
The new rules just better reflect what dotnet format has built in.